### ISSUE TITLE
extraneous output

### ISSUE THREAD

**Issue by substack** [websockets/wscat#2]
The extraneous output (colors, `>` and `<` characters) makes it very hard to use this program with other programs that expect unmodified data from stdin and stdout:

```
$ wscat -c ws://localhost:8081/sync 

connected (press CTRL+C to quit)

< 6

<0"/
+["55e06327e530a87a3f21f1f852681dc1","sync"]
> 
```

This makes it difficult for me to take the data and feed it into another program. For example, here I've got forkdb running in the browser and I would like to sync that to a forkdb running on the server using just stdin and stdout but the line numbers, colors, status logging, and decoration gets in the way of that use case.

This is very un-like netcat, which just provides a raw undecorated stream on stdin and stdout.

In the mean time, I whipped up [wsnc](https://npmjs.org/package/wsnc) but I think it would be better for this package to remove decoration.

**Comment by nwwells** [comment]
Rather, how about a raw mode, as I've found it useful to have the colors and extra output.

**Comment by nwwells** [comment]
Fixed in a2ae4f59773d5848b8a15c6dc7014ca8f8180b8c and 2de4e0f0a7341b83024cde0a3a1e7403a132f001. Will be released with 1.2.0 (see #35)

At least the output is. If streaming input into wscat would be beneficial, please open a new issue.

**Comment by nwwells** [comment]
>  If streaming input into wscat would be beneficial, please open a new issue.

Actually, #4 will cover handling stdin properly.
